website/source/downloads.html.erb

```diff
@@ -1,61 +1,14 @@
 ---
-layout: "downloads"
+layout: "inner"
 page_title: "Download Vault"
-sidebar_current: "downloads-vault"
 description: |-
   Download Vault
 ---
 
-<h1>Download Vault</h1>
-
-<section class="downloads">
-  <div class="description row">
-    <div class="col-md-12">
-      <p>
-        Below are the available downloads for the latest version of Vault
-        (<%= latest_version %>). Please download the proper package for your
-        operating system and architecture.
-      </p>
-      <p>
-        You can find the
-        <a href="https://releases.hashicorp.com/vault/<%= latest_version %>/vault_<%= latest_version %>_SHA256SUMS">
-          SHA256 checksums for Vault <%= latest_version %>
-        </a>
-        online and you can
-        <a href="https://releases.hashicorp.com/vault/<%= latest_version %>/vault_<%= latest_version %>_SHA256SUMS.sig">
-          verify the checksums signature file
-        </a>
-        which has been signed using <a href="https://hashicorp.com/security.html" target="_blank" rel="nofollow noopener noreferrer">HashiCorp's GPG key</a>.
-        You can also <a href="https://releases.hashicorp.com/vault/" target="_blank" rel="nofollow noopener noreferrer">download older versions of Vault</a> from the releases service.
-      </p>
-      <p>Check out the <a href="https://github.com/hashicorp/vault/blob/v<%= latest_version %>/CHANGELOG.md">v<%= latest_version %> CHANGELOG</a> for information on the latest release.</p>
-      <p><a href="/community.html">Community resources</a> are available to learn more about Vault and interact with the community.
-    </div>
-  </div>
-
-  <% product_versions.each do |os, arches| %>
-    <% next if os == "web" %>
-    <div class="row">
-      <div class="col-md-12 download">
-        <div class="icon pull-left"><%= system_icon(os) %></div>
-        <div class="details">
-          <h2 class="os-name"><%= pretty_os(os) %></h2>
-          <ul>
-            <% arches.each do |arch, url| %>
-              <li><a href="<%= url %>"><%= pretty_arch(arch) %></a></li>
-            <% end %>
-          </ul>
-          <div class="clearfix"></div>
-        </div>
-      </div>
-    </div>
-  <% end %>
-
-  <div class="row">
-    <div class="col-md-12 poweredby">
-      <a href="https://www.fastly.com?utm_source=hashicorp" target="_blank" rel="nofollow noopener noreferrer">
-        <%= inline_svg "fastly.svg", height: 50 %>
-      </a>
-    </div>
-  </div>
-</section>
+<hashi-product-downloader
+  product="Vault"
+  version="<%= config[:latest_version] %>"
+  date="2018-08-30"
```

Comment:
Ideally we can get these variables into `middleman-hashicorp` so we only have to update config.rb. In particular the product name and date. The build I think we'll throw out since we don't use it. But having these in one source we can update when we cut a release (`config.rb`) would be ideal.

Can be done post-merge but sooner would be better on this.

Reply:
👍